rdrf/rdrf/helpers/utils.py

```diff
@@ -705,6 +707,26 @@ def get_supported_languages():
     return [Language(pair[0], pair[1]) for pair in settings.LANGUAGES]
 
 
+LanguageInfo = namedtuple('Language', ['code', 'name'])
+
+
+def get_all_language_codes():
+    languages_in_settings = dict(settings.LANGUAGES)
+    language_codes = set(languages_in_settings.keys())
+    extra_language_codes = [language_code for language_code in LANGUAGE_ALPHA3.keys() if not any(language_code in code for code in language_codes)]
+    language_codes.update(extra_language_codes)
+    languages = []
+    if 'pseudo' in language_codes:
+        languages = [LanguageInfo('pseudo', 'pseudo')]
+        language_codes.remove('pseudo')
+    for subtag in sorted(language_codes):
+        alpha3_language_name = Language.get(standardize_tag(subtag)).autonym()
+        if 'Unknown' not in alpha3_language_name:
```

Comment:
fyi: There is a `startswith` method for strings

But this should use the `is_valid` method in the Language class before trying to get the autonym